README.md

```diff
@@ -56,7 +66,11 @@ Code-related documentation can be found in the [documentation](Documentation/) f
 
 ## How do I build and run this?
 
-See the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md). Serenity runs on Linux, macOS (aarch64 might be a challenge), Windows (with WSL2) and many other *Nixes with hardware or software virtualization.
+See the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md) or the [Ladybird build instructions](Documentation/BuildInstructionsLadybird.md).
+
+Serenity runs on Linux, macOS, Windows (with WSL2) and many other *Nixes with hardware or software virtualization.
```

Comment:
You never defined what is `Serenity`. I guess you wanted to somehow include Lagom, but I think it lacks some definition. Also an OS running on another seems quite weird to say IMO.

Reply:
Updated this section to be more explicit